### PR TITLE
Cookie Notice Update

### PR DESCRIPTION
**Pull request by jonhadley**
Pull request adds https://www.freeprivacypolicy.com/free-cookie-consent/ cookie consent dialog replacing and removing previous version

- Add references to new scripts (sgov & hs)
- Update alert JS to separate dismissal and saving of preferences for Alerts.
- Refactor behaviour where Alerts init JS is called on the page multiple times.
- Add attributes to scripts that need to not be loaded by default (not strictly-necessart, GA, SiteImprove, Alerts)
- Add style overrides for cookie banner (specifically for  stockport gov)
- Add cookie preferences link to header (sgov/hs)
- Update JS script references (hashes have changed because of updates to alerts scripts)
- Refactor HS header to make links work and improve layout
- Remove styling and controls for old cookie consent solution
- Some clean up and refactoring as I went

### CODE REVIEW

File: src/StockportWebapp/Views/Shared/CookieNotice.cshtml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+﻿    <!-- Cookie Consent by FreePrivacyPolicy.com https://www.FreePrivacyPolicy.com -->
+    <script type="text/javascript" src="//www.freeprivacypolicy.com/public/cookie-consent/4.1.0/cookie-consent.js" charset="UTF-8"></script>
+    <script type="text/javascript" charset="UTF-8">
```
Comment (elena-secara) [inline src/StockportWebapp/Views/Shared/CookieNotice.cshtml:3]: type attributes can be removed. default is text/js

File: src/StockportWebapp/Views/stockportgov/Shared/GlobalAlerts.cshtml [hunk 2]
```diff
@@ -12,4 +12,24 @@
...
+
+<script type="text/plain" data-cookie-consent="functionality">
+    require(['/assets/javascript/config-80795c2e.min.js'], function () {
+        require(['alertsSetCookie'],
+            function (alertsSetCookie) {
+                alertsSetCookie.Init();
+            }
+        );
+    });
+</script>
```
Comment (elena-secara) [inline src/StockportWebapp/Views/stockportgov/Shared/GlobalAlerts.cshtml:35]: Could you call the AlertInit partial instead?

File: src/StockportWebapp/Views/stockportgov/Shared/_LayoutSemantic.cshtml [hunk 3]
```diff
@@ -23,14 +23,17 @@
...
     <link rel=stylesheet type=text/css href=@stylesheet.StyleguideSemanticV2 />
     <link rel=stylesheet type=text/css href=@stylesheet.Slick />
-
-    <script src="~/assets/javascript/vendor/require.min.js"></Script>
-
-    <partial name="TagManager/TagManagerDataLayer" />
-    <partial name="TagManager/TagManager" />
+    <link rel=stylesheet href=~/assets/stylesheets/site-cookieNotice.min.css asp-append-version=true />
+    
+    <script type=text/javascript src=~/assets/javascript/vendor/require.min.js asp-append-version=true></script>
```
Comment (elena-secara) [inline src/StockportWebapp/Views/stockportgov/Shared/_LayoutSemantic.cshtml:28]: type attribute can be removed

**Review by elena-secara — APPROVED** [review]

File: src/StockportWebapp/Views/Shared/CookieNotice.cshtml (hunk 1 above)
Comment (jonhadley) [reply src/StockportWebapp/Views/Shared/CookieNotice.cshtml]: Done

File: src/StockportWebapp/Views/stockportgov/Shared/GlobalAlerts.cshtml (hunk 2 above)
Comment (jonhadley) [reply src/StockportWebapp/Views/stockportgov/Shared/GlobalAlerts.cshtml]: Done

File: src/StockportWebapp/Views/stockportgov/Shared/_LayoutSemantic.cshtml (hunk 3 above)
Comment (jonhadley) [reply src/StockportWebapp/Views/stockportgov/Shared/_LayoutSemantic.cshtml]: Done

**Review by elena-secara — APPROVED** [review]

**Pull request merged**